lib/utils/getImageSizes/index.ts

```diff
@@ -0,0 +1,21 @@
+// Returns a string to be used in the sizes property of the Image component
+// This function assumes that the image layout property equals fill
+// and that the image fills the viewport only if it is width is lower than 768px
+export const getImageSizes = (desktop_width: number) => {
```

Comment:
This is cool. What if we follow our universal breakpoints:
`small, medium, large, desktop, desktopLarge`

https://nextjs.org/docs/basic-features/image-optimization#local-images

we can tweak it to override the requested image size for *any* breakpoint
(otherwise default to 100vw)
```
sizes={getImageSizes({ large: "75rem", desktopLarge: "33vw" })}
```